projects/kit/components/data-list-wrapper/data-list-wrapper.component.ts

```diff
@@ -55,17 +60,17 @@ export class TuiDataListWrapperComponent<T> implements TuiDataListAccessor<T> {
 
     constructor(
         @Inject(TUI_ITEMS_HANDLERS)
-        private readonly itemsHandlers: TuiItemsHandlers<T>,
+        private readonly itemsHandlers: TuiItemsHandlers<T | T[]>,
     ) {}
 
     @Input()
     @tuiDefaultProp()
-    itemContent: PolymorpheusContent<TuiValueContentContext<T>> = ({$implicit}) =>
+    itemContent: PolymorpheusContent<TuiValueContentContext<T>> | null = ({$implicit}) =>
```

Comment:
Why `null`? I want to get rid of `null` in all content because empty string can act as empty content without joint type.

Reply:
fixed